torchvision/prototype/transforms/_geometry.py

```diff
@@ -125,20 +80,16 @@ def __init__(
         if (scale[0] > scale[1]) or (ratio[0] > ratio[1]):
             warnings.warn("Scale and ratio should be of kind (min, max)")
 
-        # Backward compatibility with integer value
-        if isinstance(interpolation, int):
-            warnings.warn(
-                "Argument interpolation should be of type InterpolationMode instead of int. "
-                "Please, use InterpolationMode enum."
-            )
-            interpolation = _interpolation_modes_from_int(interpolation)
-
         self.size = size
         self.scale = scale
         self.ratio = ratio
         self.interpolation = interpolation
+        self.antialias = antialias
 
     def _get_params(self, sample: Any) -> Dict[str, Any]:
+        # vfdev-5: techically, this op can work on bboxes/segm masks only inputs without image in samples
+        # What if we have multiple images/bboxes/masks of different sizes ?
+        # TODO: let's support bbox or mask in samples without image
```

Comment:
Correct. This logic was added before you land kernels for handling crops BBoxes and masks. Now we can review this transform. It's worth noting that passing bboxes or masks without image doesn't make a lot of sense in practice because this will cause desyncing between the image and the targets. This is something that was brought up many times as a potential feature (passing only bboxes) but so far we haven't found a compelling example on where this can be used in practice. Happy to review this if we do, but for now we can assume that an image (or video coming up) will be available to give us access to the width/height here.

Reply:
Not a practical use-case, but we can't test this transform on bboxes/masks without an image :)

Reply:
I'm curious to see if this creates issues such as having no access to specific meta-data you would need from the image. If you can do this without them, we can chat adding it. But if you have to add lots of workarounds to support it, we should find a real-world use-case. I think the test use-case can be achieved by passing an equivalent image.